bond-vlan-pre.sh

```diff
@@ -17,6 +17,6 @@
 #
 # Red Hat Author(s): Radek Vykydal <rvykydal@redhat.com>
 
-TESTTYPE="network rhbz1903061"
+TESTTYPE="network fedora-only"
```

Comment:
Can you please add a comment pointing to https://bugzilla.redhat.com/show_bug.cgi?id=1879480 so that the "fedora-only" has some explanation attachhed? Supposedly that will trickle into rhel 9, so we can enable it then. Or perhaps it would make sense to have a `not-rhel8` tag, so that it'll automatically run on RHEL 9? (That can be a follow-up, though)

Reply:
Sure, I like the idea of not-rhel8. I should do similar thing in https://github.com/rhinstaller/kickstart-tests/pull/434.